test/index.test.js

```diff
@@ -723,16 +750,16 @@ test('can override stream payload content-length header', (t) => {
   const headers = { 'content-length': '100' }
 
   inject(dispatch, { method: 'POST', url: '/', payload: getTestStream(), headers }, (err, res) => {
-    t.error(err)
-    t.equal(res.payload, '100')
+    t.equal(err instanceof inject.errors.ContentLength, true)
+    t.error(res)
```

Comment:
The test says `can override stream payload content-length header` - are we blocking this feature now?

Reply:
Yes
1. it is not supported with test example http.Server/http.request
2. I can't call the Dispatch function unless I read everything from Readable